[refactor] remove redundant this

1000xCoder opened the pull request:
# Proposed changes

Removing redundant this, since it is required mostly in constructor.

Issue Number: #9775

## Problem Summary:

### Before pr: 

```
public AlterHandler getSchemaChangeHandler() {
        return this.schemaChangeHandler;
}
```

### After pr:

```
public AlterHandler getSchemaChangeHandler() {
        return schemaChangeHandler;
}
```

## Checklist(Required)

1. Does it affect the original behavior: No
2. Has unit tests been added: No
3. Has document been added or modified: No Need
4. Does it need to update dependencies: No
5. Are there any changes that cannot be rolled back: No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

Commit:
remove redundant this

Commit:
remove redundant this

jackwener commented on fe/fe-core/src/main/java/org/apache/doris/backup/AbstractJob.java:
```diff
@@ -87,7 +87,7 @@ protected AbstractJob(JobType type, String label, long dbId, String dbName,
         this.label = label;
         this.dbId = dbId;
         this.dbName = dbName;
-        this.createTime = System.currentTimeMillis();
```
I think `this.` in constructor can remain.
Because I think it's better to keep same style.

jackwener commented on fe/fe-core/src/main/java/org/apache/doris/backup/BackupJob.java:
```diff
@@ -124,7 +124,7 @@ public BackupJob(String label, long dbId, String dbName, List<TableRef> tableRef
                      BackupContent content, Catalog catalog, long repoId) {
         super(JobType.BACKUP, label, dbId, dbName, timeoutMs, catalog, repoId);
         this.tableRefs = tableRefs;
-        this.state = BackupJobState.PENDING;
```
same with above.

jackwener commented on fe/fe-core/src/main/java/org/apache/doris/backup/Repository.java:
```diff
@@ -119,7 +119,7 @@ public Repository(long id, String name, boolean isReadOnly, String location, Blo
         this.isReadOnly = isReadOnly;
         this.location = location;
         this.storage = storage;
-        this.createTime = System.currentTimeMillis();
```
same with above.

Commit:
add this in constructor

Commit:
add this in constructor

Commit:
remove redundant this

Commit:
remove redundant this

Commit:
remove redundant this

jackwener commented on fe/fe-core/src/main/java/org/apache/doris/task/AgentBatchTask.java:
```diff
@@ -64,7 +64,7 @@ public class AgentBatchTask implements Runnable {
 
     public AgentBatchTask() {
-        this.backendIdToTasks = new HashMap<Long, List<AgentTask>>();
+         backendIdToTasks = new HashMap<Long, List<AgentTask>>();
```
redundant space

jackwener commented on fe/fe-core/src/main/java/org/apache/doris/task/AgentBatchTask.java:
```diff
@@ -89,8 +89,8 @@ public void addTask(AgentTask agentTask) {
         List<AgentTask> tasks = new LinkedList<AgentTask>();
-        for (Long backendId : this.backendIdToTasks.keySet()) {
-            tasks.addAll(this.backendIdToTasks.get(backendId));
+        for (Long backendId :  backendIdToTasks.keySet()) {
+            tasks.addAll( backendIdToTasks.get(backendId));
```
same with above

jackwener reviewed (commented):
Thanks for your contribution.❤
It's a job need some patient😂.

jackwener commented:
BTW, please git rebase and fix the conflict

Commit:
fix spacing

Commit:
fix spacing

Commit:
resolve conflict

Commit:
resolve conflict

Commit:
resolve conflict

jackwener commented:
You can learn knowledge about git.
You can use `git rebase` and `fix conflict`.

Now, this PR include some other PR....😂
![image](https://user-images.githubusercontent.com/30525741/170852280-45e6e0bf-2e86-44fe-9e31-449a2fcd0022.png)

jackwener commented:
BTW, thought this problom of `this.` exist in many files.

You can just fix 10-20 files rather than so many files which is easier to the first-time contribution.

1000xCoder committed:
Merge branch 'apache:master' into remove-redundant-this

Vikasss7663 commented:
I am trying this.

yixiutt committed:
[improvement](load) reduce useless err_msg format in VOlapTableSink send (#9531)

stalary committed:
[fix](sql-block-rule) sql block rule NPE (#9778)

compiletheworld committed:
[fix](third-party) Fix build issues of third-party in ubuntu docker (#9805)

Fixes:
1. Fully static linkage of protoc may lead to core dump when using it.
2. Building libmyclient may fail due to miss-used option `-v` of `make`
3. Building lzma implicitly requires autopoint
4. Building xml2 implicitly requires pkg-config

morrySnow committed:
[enhancement] (community) update intellij style file (#9783)

nextdreamblue committed:
[bugfix] Fix create table like when having hidden columns (#9694)

jackwener commented:
😂 new conflict

amosbird committed:
[Bugfix(Vec)] Fix some memory leak issues (#9824)

Gabriel39 committed:
[Improvement] Simplify expressions for _vconjunct_ctx_ptr (#9816)

smallhibiscus committed:
[doc]Fix typo in create-table documentation. (#9834)

Co-authored-by: smallhibiscus <844981280>

smallhibiscus committed:
[regression test]Add regression test case for  multiple column delete. (#9817)

Co-authored-by: smallhibiscus <844981280>

EmmyMiao87 committed:
[API changed](parser) Remove merge join syntax (#9795)

Remove merge join sql and merge join node

sumtumn committed:
[feature](function) Add nvl function (#9726)

dujl committed:
[feature](hudi) Step2: Support query hudi external table(include cow and mor table) (#9752)

support query cow and mor hudi table.

smallhibiscus committed:
[improvement](script) Add jvm parameters and the process will automatically stop when oom occurs in fe. (#9765)

morningman committed:
[community] add more collaborators (#9830)

Add new collaborators:
    - dataalive

yiguolei committed:
[Improvement] optimize  scannode concurrency query performance in vectorized engine. (#9792)

smallhibiscus committed:
[doc]Modify export example in en and zh-cn doc. (#9840)

Co-authored-by: smallhibiscus <844981280>

zy-kkk committed:
[test]Add schema change for regression test (#9836)

* add alter table column test

* add test date

* add cancelled state judge

JNSimba committed:
[doc]add alter resource doc (#9837)

Co-authored-by: wudi <>

zy-kkk committed:
[test]add update test (#9829)

jackwener committed:
[enhancement](style): polish the `checkstyle` (#9767)

* [enhancement](style): cancel auto tab for Javadoc.

* *: fix review

adonis0147 committed:
[refactor][be] Optimize headers (#9708)

amosbird committed:
[Bugfix(Vec)] Close result_sink properly (#9849)

Close result_sink properly so that error code is reported and
expr_context is always closed.

Gabriel39 committed:
[BUG] return NULL for invalid date value (#9862)

hf200012 committed:
[fix][doc]Doc fix (#9851)

doc fix

JNSimba committed:
[doc] fix alter resource doc style (#9866)

morningman committed:
[feature-wip](multi-catalog) Add basic class and interface for multi catalog support

HappenLee committed:
[Opt][VecLoad] Opt the vec stream load performance (#9772)

Co-authored-by: lihaopeng <lihaopeng@baidu.com>

HappenLee committed:
[Bug][Fix] One Rowset have same key output in unique table (#9858)

Co-authored-by: lihaopeng <lihaopeng@baidu.com>

Vikasss7663 commented:
What should I do now ?
I am not able to resolve that conflict issue.
The one thing now I can do is again create fork and starting all over again.

Vikasss7663 commented:
If you have **good suggestions** than your welcome.

Vikasss7663 commented:
are you talking about 
fe/fe-core/src/main/java/org/apache/doris/blockrule/SqlBlockRule.java
this files conflict.

Vikasss7663 commented:
This is what showing

![Uploading Screenshot 2022-05-31 at 10.55.59 AM.png…]()

jackwener commented:
> The one thing now I can do is again create fork and starting all over again.

 😂It's ok. A PR with a lot of files has conflicts isn't easy for the newer.

You can create a new PR.

What's more, you also can learn something about `git rebase` and fix conflict, which is useful to your next open-source trip.

Commit:
remove redundat this SqlBlockRule

jackwener commented:
usually, my behavior.

```bash
# set doris upstream and fetch
git remote add upstream https://github.com/apache/incubator-doris.git
git fetch --all

# rebase to upstream master
git rebase upstream/master

# fix conflict
```

Commit:
remove redundat this SqlBlockRule

jackwener commented:
Some material introduce it. like this 

https://www.youtube.com/watch?v=RGtwxYqkkas

Vikasss7663 commented:
Now it's ok or not ?

Or I have to create new PR

jackwener commented:
> Now it's ok or not ?
> 
> Or I have to create new PR

You can create a new PR, it's ok and easier

jackwener commented:
You can try
```
git rebase 

git push -f
```

in new PR.

It will be better than `git merge`, which can avoid following.

![image](https://user-images.githubusercontent.com/30525741/171101553-3f489de3-36f2-446e-accc-49241c3da504.png)

jackwener commented:
I forgot it can't comment after close PR

Vikasss7663 commented:
When I am trying to create new pull request 

<img width="1396" alt="Screenshot 2022-05-31 at 11 27 12 AM" src="https://user-images.githubusercontent.com/98081274/171102871-d4f3c8f0-8799-4652-aea9-58b0f000fd10.png">

jackwener commented:
You can recreate a new branch.
Carry this PR's modification in about 10 files to it.

This branch after `git merge` is a little mess.
